### ISSUE TITLE
Installing strongswan apt package on ubuntu breaks calico node in microk8s

### ISSUE THREAD

**Issue by turowicz** [projectcalico/calico#4744]
Scenario: I am trying to move my client servers to L2TP IPsec VPN with `network-manager-l2tp` package, which has a dependency on `strongswan` package.

After installation the microk8s (with calico stack) dies terribly.

Details here: https://github.com/ubuntu/microk8s/issues/2410

**Comment by turowicz** [comment]
cc @lwr20 @fasaxc @mgleung @champtar @lmm

**Comment by caseydavenport** [comment]
@turowicz sorry for missing this - sounds like, from the linked issue, that this is effectively breaking Kubernetes services or just breaking host networking.

I think the first step would be to determine what does work - can your node communicate directly with the control plane / API server nodes via IP? Can it do so via a Service?

**Comment by turowicz** [comment]
I can access the control plane but the pods in other namespaces are unable to do any networking at all.

**Comment by turowicz** [comment]
cc @caseydavenport

**Comment by lwr20** [comment]
> I can access the control plane with kubectl but the pods in are unable to do any networking at all.

That's not quite what Casey was asking.  If you ssh into the nodes, can they talk to the kubernetes apiserver service?  Can they talk to the control plane node IPs directly?  Can they talk to each other?

**Comment by turowicz** [comment]
@lwr20 it's a single node scenario

**Comment by caseydavenport** [comment]
Afraid we need a bit more to go on here - what is it that strongswan is doing that is breaking the cluster?

**Comment by turowicz** [comment]
I am trying to set up a VPN. Strongswan is a dependency apt package.

**Comment by caseydavenport** [comment]
Yes sorry, I understand that, but we need to know where the networking is breaking down. You'll need to dig in to see if strongswan is, for example, changing routes, changing firewall policy on the host in some way, or something else, that is disrupting Calico.

**Comment by turowicz** [comment]
As we are dropping microk8s in favour of K3S the problem no longer exists for us.
